airbyte-webapp/src/components/ui/Switch/Switch.module.scss

```diff
@@ -112,6 +114,10 @@
     width: 0;
     height: 0;
 
+    &:focus-visible ~ .slider {
+      @include mixins.focus-ring-style;
```

Comment:
here's where it's a little funky.

I attempted adding an optional argument to the mixin where we could pass a class and therefore apply the mixin directly to .switch, but spiraled.  Can take another stab if there's a strong feeling that we should do that!

Reply:
I think putting this inline here sounds like the better idea, than overengineering the mixin for potentially very rare use-cases :)